config/environments/development.rb

```diff
@@ -1,4 +1,6 @@
-require "active_support/core_ext/integer/time"
+# frozen_string_literal: true
+
+require 'active_support/core_ext/integer/time'
 
 Rails.application.configure do
```

Comment:
Please enable mode for:
1) missing locales
2) raise on missing translations
3) deprecation warning log to console